test/future_execution_test.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require_relative 'test_helper'
+require 'multi_json'
```

Comment:
This doesn't seem necessary :thinking:

Reply:
I needed to do this when running tests only in this file, will drop.